.github/workflows/test.yml

```diff
@@ -162,3 +162,24 @@ jobs:
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
       - name: Image digest
         run: echo ${{ steps.docker_build.outputs.digest }}
+
+  autodeploy:
+    runs-on: ubuntu-latest
+    needs: [docker-deploy]
+    if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/develop' || startsWith(github.ref, 'refs/tags/')
+    steps:
+    - uses: actions/checkout@v2
+    - name: Deploy Staging
+      if: github.ref == 'refs/heads/main'
+      run: bash scripts/autodeploy.sh
+      env:
+        AUTODEPLOY_URL: ${{ secrets.AUTODEPLOY_URL }}
+        AUTODEPLOY_TOKEN: ${{ secrets.AUTODEPLOY_TOKEN }}
+        TARGET_BRANCH: "staging"
```

Comment:
Wouldn't `IMAGE_TAG` or similar be more representative of this field? (we don't have a branch called `staging`)

Reply:
I'll change it, this comes from other autodeploy projects I didn't want to change the var just because Devops team may be familiarized but completely agree, another could be better.